ddtrace/tracer.py

```diff
@@ -70,6 +70,14 @@
 _INTERNAL_APPLICATION_SPAN_TYPES = {"custom", "template", "web", "worker"}
 
 
+def get_partial_flush_enabled():
```

Comment:
Should probably add these as options to `configure` for those who can't use env vars